WordPress/src/main/java/org/wordpress/android/ui/media/WordPressMediaUtils.java

```diff
@@ -260,6 +261,29 @@ public static void loadNetworkImage(String imageUrl, NetworkImageView imageView,
         }
     }
 
+    public static void loadNetworkImage(String imageUrl, WPNetworkImageView imageView) {
```

Comment:
Yes, the other one will be gone soon (I hope) once https://github.com/wordpress-mobile/WordPress-Android/issues/6296 is done.